apps/css-workshop/src/components/Anchor.astro

```diff
@@ -0,0 +1,18 @@
+---
+type Props = {
+  external?: 'true';
+  status?: 'informational' | 'positive' | 'warning' | 'negative';
+  underline?: 'true';
+} & astroHTML.JSX.HTMLAttributes;
+
+const { class: className, external, href, status, underline, ...props } = Astro.props;
+---
+
+<a
+  class:list={['iui-anchor', className, external ? 'iui-anchor-external' : '']}
+  data-iui-status={status}
+  data-iui-underline={underline}
+  {...external ? { referrerpolicy: 'no-referrer', target: '_blank' } : {}}
+  href={href ? href : '#'}
```

Comment:
We shouldn't have such hacks here. Better to have the consuming components manually add `href='#'` because it's an anti-pattern

Reply:
Removed and added `href="#"` to individual demos.